### DISCUSSION TITLE
how to separate tensorboard dir and log dir

### DISCUSSION THREAD

**Discussion by jaffe-fly** [NVIDIA-NeMo/NeMo#2931] [Q&A] [answered]
i found exp_manager.exp_dir will save log and tensorboard together.i want to separate  save in diff dir ,how to set ?

**Comment by VahidooX** [comment, marked as answer]
We don't have that feature, but you may have two options:

+change the code here and pass another path to the loggers: https://github.com/NVIDIA/NeMo/blob/main/nemo/utils/exp_manager.py#L283

+create a symbolic link for the checkpoint folder at the same path (which points to somewhere else) so that the checkpoints can be saved in another place.

**Reply by jaffe-fly** [reply]
the second maybe good for me
